src/Illuminate/Queue/Queue.php

```diff
@@ -341,12 +341,13 @@ protected function shouldDispatchAfterCommit($job)
      *
      * @param  string|int|null  $jobId
      * @param  \Closure|string|object  $job
+     * @param  string  $payload
      * @return void
      */
-    protected function raiseJobQueuedEvent($jobId, $job)
+    protected function raiseJobQueuedEvent($jobId, $job, $payload)
```

Comment:
This is _technically_ a breaking change, however I'm not sure if we expect anyone to override it.

If we are concerned about this, we use magic and just access it via `func_get_args()` but not have it in the method signature. We have done this before in the framework.